lib/hardware_interface/fcb_messages/fcb_message_definitions.dart

```diff
@@ -102,22 +140,43 @@ Map<String, Object> parseMessage(int packetType, ByteData data) {
     var packetObject = MessageOptions.options[packetType];
     var parsedBinary = parseData(data, packetObject!.binaryFormatString);
     messageDict["message_type"] = packetObject.messageType;
+    var messageType = packetObject.messageType;
 
     for (var i = 0; i < packetObject.parameters.length; i++) {
       var parameter = packetObject.parameters[i];
       var value = parsedBinary[i];
+      var databaseKey = parameter.databaseKey;
+
+      if (parameter.databaseKey == "pyro_continuity") {
+        if (messageDict["message_type"] == "Pyro Info") {
+          var pyroContinuity = value.toRadixString(2);
+          value = pyroContinuity;
+          var padLeftZeros = 8 - pyroContinuity.length;
```

Comment:
Ideally, we'd be able to extract the `8` width from the `PyroInfo` message data definition. If that's difficult though probably worth making it a named constant here at least.

Reply:
Instead of string parsing is something like `(1 << i) & pyroContinuity` make the intent here more clear?

Reply:
I did bitwise math with @mcm001 check